test/function_compilation.jl

```diff
@@ -187,4 +189,38 @@ using DataFramesMeta
         end
     end
 end
+
+@testset "composed compilation" begin
+    @eval begin
+        df = DataFrame(a = [1], b = [2])
+
+        f(x) = identity(x)
+        g(x, y) = x + y
+
+        df_wide = DataFrame(rand(10, 1000), :auto)
+    end
+
+    for _ in 1:2
+        @eval begin
+            @test @select(df, :y = (f ∘ g)(:a, :b)).y == [3]
```

Comment:
Do you think testing timing is enough? Maybe also generated code should be tested?

Reply:
We don't check generated code anywhere else. I think this should be enough.

Reply:
Nevermind it's easy to add. Added. Will merge after tests.